shared/pub/metadata.js

```diff
@@ -49,3 +49,49 @@ export const getTextAbstract = (docJson) => {
 	}
 	return abstract;
 };
+
+export const getGSNotes = (notes) => {
```

Comment:
Could we call this `getGoogleScholarNotes` instead? `GS` is a little opaque for someone reading this code out of context.

Reply:
+1